requirements/dev.txt

```diff
@@ -0,0 +1,2 @@
+pipenv
```

Comment:
It would make sence now if  `pipenv` and `pre-commit` are supposed to be installed before installing the project.
I think it could be reasonable now to exclude them  in the requirements folder.

I agree with you that there is no need to keep the version consistency